uninstall.php

```diff
@@ -28,6 +28,24 @@
  * and to ensure only the site owner can perform this action.
  */
 if ( Constants::is_true( 'WC_REMOVE_ALL_DATA' ) ) {
+	/**
+	 * Load core packages autoloader.
+	 */
+	if ( version_compare( PHP_VERSION, '5.6.0', '>=' ) ) {
```

Comment:
Maybe this check is not needed anymore now that WooCommerce core requires PHP >= 7.0?